salt/transport/client.py

```diff
@@ -78,7 +78,7 @@ class AsyncChannel(object):
     _resolver_configured = False
 
     @classmethod
-    def _config_resolver(cls, num_threads=10):
+    def _config_resolver(cls, num_threads=1):
```

Comment:
This new default affects both Windows and non-Windows. However, you only changed the Windows logic above. So you are introducing a logical difference in Linux.

Reply:
My change actually only affects Linux (because on NILinuxRT we disable multiprocessing). And event the memory leak itself only reproduced on Linux . See https://bugs.python.org/issue44871. 
What you see above is making sure I don't break windows by making this change, since the process var won't be initialized anymore.

However, this change affects the number of threads that will be used to resolve the master ip addres from the dns name. (https://www.tornadoweb.org/en/stable/netutil.html#tornado.netutil.ThreadedResolver) Given the fact that there's always only one master I think 1 thread should be enough :)